### PR TITLE
[COMMISSION] Adds Til the Last Shot

### PR DESCRIPTION
**Pull request by Kitsunemitsu**
## About The Pull Request
Adds 'Til the Last Shot, a spreading meat abnormality.

image

Still needs records

## Changelog
:cl:
add: Added Til the Last Shot
/:cl:

### CODE REVIEW

**Review by Coxswain-Navigator — COMMENTED** [review]
Nothing strikes me as fundamentally flawed, I would like to see it finished.

**Comment by JackFrost2424** [comment]
From the test we did last round: 
>Meatvines have been proven to basically be hard to break even with high damage weapons. Might be intentional but considering how annoying they are, might not be a good idea 
>14 boxes? Kinda weird for a HE of all things. I would at least make it 16-18. 
>I would drastically reduce the HP of the meatmen. Maybe to 250-300.  And lower the damage accordingly. Considering how quickly they can spawn. Make the projectile do 20 and make their melee have 10 at the low end.
>EGO is fine if the other changes were done.

File: code/modules/paperwork/records/info/aleph.dm [hunk 1]
```diff
@@ -165,3 +165,18 @@
...
+
+//Last Shot
+/obj/item/paper/fluff/info/aleph/last_shot
+	abno_type = /mob/living/simple_animal/hostile/abnormality/last_shot
+	abno_code = "C-06-152"
+	abno_info = list(
+		"Employees with low Temperance had their work chance increased dramatically.",
+		"Employees with Justice or Fortitude under level 5 recieved more work damage.",
+		"When the work result was Bad, the Qliphoth Counter lowered.",
+		"When the Qliphtoh Counter reached 0, last shot bred flesh until it was destroyed in it's containment.")
```
Comment (LanceSmites328) [inline code/modules/paperwork/records/info/aleph.dm:179]: Does not mention high temperance reducing success chance. While it seems redundant, the Justice and Fortitude requirements should be stated on two separate lines to indicate their stacking nature.
```suggestion
	abno_info = list(
		"Employees with low Temperance had their work chance increased dramatically.",
		"Employees with high Temperance had their work chance reduced dramatically.",
		"Employees with Fortitude under level 5 received more work damage.",
		"Employees with Justice under level 5 received more work damage.",
		"When the work result was Bad, the Qliphoth Counter lowered.",
		"When the Qliphtoh Counter reached 0, last shot bred flesh until it was destroyed in it's containment.")
```

**Review by LanceSmites328 — CHANGES_REQUESTED** [review]

File: code/modules/paperwork/records/info/aleph.dm [hunk 2]
```diff
@@ -165,3 +165,20 @@
...
+
+
+
+//Last Shot
+/obj/item/paper/fluff/info/aleph/last_shot
+	abno_type = /mob/living/simple_animal/hostile/abnormality/last_shot
+	abno_code = "C-06-152"
+	abno_info = list(
+		"Employees with low Temperance had their work chance increased dramatically.",
+		"Employees with high Temperance had their work chance reduced dramatically.",
```
Comment (LanceSmites328) [inline code/modules/paperwork/records/info/aleph.dm:177]:
```suggestion
		"Employees with Temperance levels below 4 had their work chance increased dramatically.",
		"Employees with Temperance levels 5 and above had their work chance reduced dramatically.",
```

**Review by LanceSmites328 — CHANGES_REQUESTED** [review]

**Review by LanceSmites328 — APPROVED** [review]

**Pull request merged**